main.tex

```diff
@@ -19,5 +19,29 @@
 
 \maketitle
 
+% 1.2.1
+\subsubsection{Facebook}
+Es wurde gezeigt, dass Facebook 2011/2012 eine große Rolle in der gesellschaftlichen Entwicklung einnahm \parencite{empowermentThroughFacebook}.
+Dies ging einher mit einer Regulierung im Jahre 2012, welche das versenden von SMS auf 200 pro Tag beschränkte \autocite{PressInformation}, in Folge dessen das Interesse an Facebook stark anstieg, da Handybesitzer weiterhin unlimitiert mit ihren Freunden kommunizieren wollten.
+Durch das Kennenlernen und Nutzen von Facebook entwickelte sich die indische Gesellschaft, welche bislang von der Globalisierung durch das Internet ausgeschlossen war, nun erstmals dahin eine gemeinsame, transnationale Identität bilden.
+\medskip
+Die immernoch währende Kastengesellschaft und strenge Überwachung von Frauen in Indien führt dazu, dass 2015 von den 135 Millionen aktiven Facebook Nutzern gerademal 25\% weiblich waren\autocite{slideshareIndia}. 
+Das weitverbreitete Vorurteil, die Keuschheit von jungen Frauen sei gefährdet, wenn sie Zugang zu Mobilgeräten hätten, macht es für diese geradezu unmöglich sich frei im Internet entfalten zu können, oder gar, ein Handy zu besitzen.
+Selbst verheirateten Frauen, welche mit Erlaubnis des Mannes ein Handy besitzen, wird nicht selten der Zugang zum Internet von den Schwiegereltern überwacht.
+Somit entwickelte sich Facebook zu einem Sozialem Netzwerk, welches durch seine maskuline Dominanz für Frauen als gefährlich erachtet wird.
+Dadurch wird oft selbst Frauen, welche Internetzugang genießen, die Nutzung Facebooks von der Familie untersagt. 
+
+
+% cut
+Durch die hohe männliche Nutzeranzahl entstand schnell ein Konkurenzkampf der Selsbtdarstellung.
+So wird der aktuelle Bildungsstand, der soziale Hintergrund und oft auch das Profilbild besser dargestellt.
+Der wirkliche soziale Stand in der Gesellschaft spielte nun im Internet kaum noch eine Rolle.
+
+Da selbst gebildete Inder kaum mit der englischen Sprache vertraut waren, wurde das Erlernen und Lehren Anderer von der `Sprache des Internets" allgegenwärtig. % cut
+So bildeten sich Lerngruppen, welche sich in Internetcafe's trafen, um dort nicht nur Wissen, sondern auch den vergleichsweise teuren Datentarif zu teilen.
+
+
+Neben den hohen Datentarifen bleib auch die Internetgeschwindigkeit ein Problem, mit welcher die Inder eine durchschnittliche Seitenladezeit von 3,9 Sekunden hatten\autocite{mashable}. Facebook versuchte auf dieses Problem mit der Messenger App einzugehen, in welcher nur Chats geladen werden, was aber nicht die endgültige Lösung darstellte. Als eine weiterer Lösungsansatz gilt das Konzept für Internet.org.
+
```

Comment:
Facebook erlebte ab 2011 ein starkes Wachstum an Nutzern in Indien, was 2012 mitunter durch die Beschränkung des Versendens von SMS auf maximal 200 pro Tag durch die TRAI unterstützt wurde, da Handybesitzer weiterhin unlimitiert kommunizieren wollten.


Dieses Kennenlernen und Nutzen von Facebook brachte den Teil der indische Gesellschaft, welcher bislang von der Globalisierung des Internets ausgeschlossen war, erstmals zur Bildung einer gemeinsamen, transnationalen Identität, was im Kontrast zur immer noch währenden Kastengesellschaft stand und steht.
Jene verwähren besonders Frauen den Zugriff auf soziale Medien und das Internet allgemein, die hohe Anzahl an Männern unter den Nutzern sei eine Gefahr für deren Keuschheit, doch selbst bei verheirateten Frauen wird häufig der Internetgebrauch durch zum Beispiel die Schwiegereltern überwacht.

\medskip

Besagte Überrepräsentation von jungen Männern zog unter anderem die Verbreitung der Misrepräsentation der eigenen Person online nach sich, was immer mehr zur Norm wurde. Dies beinhaltet zumeist den Bildungsstand, den sozialen Hintergrund und Profilbilder. So wird ein Uniabbruch plötzlich zum bestandenen Bachelor, ein solcher zum sogennanten `Honours degree', eine höher gestellte Variante.

\medskip

Das Internet brachte jedoch auch Positives mit sich: da selbst gebildete Inder kaum mit der englischen Sprache vertraut waren, wurde das Erlernen und Lehren Anderer von der `Sprache des Internets" allgegenwärtig.
So bildeten sich Lerngruppen, welche sich in Internetcafe's trafen, um dort nicht nur Wissen, sondern auch den vergleichsweise teuren Datentarif zu teilen.

Neben diesen bleib auch die Internetgeschwindigkeit ein Problem, mit welcher die Inder eine durchschnittliche Seitenladezeit von 3,9 Sekunden hatten\autocite{mashable}. Facebook versuchte auf dieses Problem mit der Messenger App einzugehen, in welcher nur Chats geladen werden, was aber nicht die endgültige Lösung darstellte. Als eine weiterer Lösungsansatz gilt das Konzept für Internet.org


%rework c: